common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,121 @@
+"""OLI analytics service event tracker backend."""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from requests_oauthlib import OAuth1Session
+
+from django.contrib.auth.models import User
+
+from student.models import anonymous_id_for_user
+from track.backends import BaseBackend
+
+
+LOG = logging.getLogger(__name__)
+
+
+class OLIAnalyticsBackend(BaseBackend):
+    """
+    Transmit events to the OLI analytics service
+    """
+    def __init__(self, **kwargs):
+        super(OLIAnalyticsBackend, self).__init__(**kwargs)
+
+        # the oli analytics service endpoint will be at
+        # "{}{}".format(oli_analytics_service_url, path)
+        self.url = kwargs.get('url', None)
+        self.path = kwargs.get('path', None)
+
+        # shared key for OLI analytics service
```

Comment:
These comments are superfluous and can be removed; they add no new
insights to the code.

It should be obvious to anyone reading this that:
- `OLIAnalyticsBackend.key` is the `... key for OLI analytics service`
- `OLIAnalyticsBackend.secret` is the `... secret for OLI analytics service`